README.md

```diff
@@ -1,4 +1,8 @@
 Citadel
 =======
 
-A turn based strategy game based on the Anura engine
\ No newline at end of file
+A turn based strategy game based on the Anura engine
+
+For support, to meet the player community and find worthy opponents for multiplayer, drop by the Discord server 
+[![Discord Server Registration](https://img.shields.io/badge/discord-server_registration-green.svg?longCache=true&style=popout&logo=discord)](https://discord.gg/Y5e3phv)
```

Comment:
More than 97% of the joins at the Discord server have been through https://discord.gg/0s6mSoZ1vJPlvcIw. Are you sure you want to propose https://discord.gg/Y5e3phv?

I don't find nothing wrong about it, but for me marks the line between merge and pass, because it turns organizational without any need for it.

Reply:
I didn't remember the original join link, so I just created a new one. I'll just change it to [the most commonly used one](https://discord.gg/0s6mSoZ1vJPlvcIw)